CHANGELOG.md

```diff
@@ -7,20 +7,28 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [7.0.0] - 2020-07-07
+
+### Added
+
+- Add `Params.RegistryMirrors` allowing to configure docker registry mirrors.
+
+### Changed
+
+- Fail if all images do not have the same registry.
+
 ## [6.4.0] - 2020-07-06
 
 ### Added
 
 - Add back registry domain configuration.
-- Add `Params.RegistryMirrors` allowing to configure docker registry mirrors.
```

Comment:
I haven't been keeping up--were these changes in 6.4.0 reverted and moved into a separate major release?

Reply:
It was re-added yes.

Reply:
But here I just made a bad job with merging changelogs in my previous PR.